io.catenax.manufacturing_capability/RELEASE_NOTES.md

```diff
@@ -2,7 +2,19 @@
 
 All notable changes to this model will be documented in this file.
 
-## [Unreleased]
+## [3.0.0]
```

Comment:
you can add today's date here if you want MS3 today.

Reply:
Done in https://github.com/eclipse-tractusx/sldt-semantic-models/pull/574/commits/3b800d833ecc8ab27b0417be2563b0bf48f0420c